src/plugins/packages.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const fs = require("fs");
+const colors = require("colors/safe");
+const Helper = require("../helper");
+const themes = require("./themes");
+
+const customCss = [];
```

Comment:
Can we simply call this `stylesheets` (and all the other occurances of `customCss`)

Reply:
Yeah, sir

Reply:
Done